mock-rollout/src/App.tsx

```diff
@@ -0,0 +1,15 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
+/* eslint-disable @typescript-eslint/no-redeclare */
+import React, { FC, useState } from 'react';
+import '@progress/kendo-theme-default/dist/all.css';
+import './App.css';
+import { ProductsList } from './Products/ProductsList/ProductsList';
+
+export const App: FC = () => {
+  return (
+    <div className="App">
+        <ProductsList />
```

Comment:
the requirements refer to this as hardware list, so the model & component naming should respect the described functionality, because it will get confusing when it will be extended 

this apply to every place where Product naming is used